src/components/tasks/card/card.module.scss

```diff
@@ -142,7 +142,7 @@
 }
 
 .progressContainerUpdated {
-    width: 12rem;
+    width: 18rem;
```

Comment:
does this hard coded value won't affect in other screen size

Reply:
It shouldn't I have checked. But let me check again.

Reply:
Removed these, still working fine.